components/predict-form/Predict.tsx

```diff
@@ -135,59 +66,10 @@ export function Predict({
   small,
   smartSetDates = true,
 }: PredictProps) {
-  const [questionType, setQuestionType] = useState<QuestionType>(
-    QuestionType.BINARY,
-  )
-
   const nonPassedRef = useRef(null) // ref must be created every time, even if not always used
   textAreaRef = textAreaRef || nonPassedRef
 
-  // TODO: move this somewhere else
-  function usePredictForm(questionType: QuestionType) {
-    const form = useForm<PredictFormType>({
-      mode: "all",
-      resolver: zodResolver(unifiedPredictFormSchema),
-    })
```

Comment:
Moved (largely unchanged) to `components/predict-form/PredictProvider.tsx`. The changed in the moved version are:
- Renaming `setQuestionType` to `questionTypeChangedCallback`, and not returning it as it wasn't used
- Adding `questionInFocus`, `setQuestionInFocus` return values